src/DrupalDoctrineCache.php

```diff
@@ -26,82 +54,63 @@ public function fetch($id) {
   }
 
   /**
-   * Checks if the Drupal cache contains the specified entry.
-   *
-   * @inheritdoc
+   * {@inheritdoc}
    */
-  public function contains($id) {
-    $entry = $this->fetch($id);
+  protected function doContains($id) {
+    $entry = $this->doFetch($id);
     return !empty($entry);
   }
 
   /**
-   * Saves data to the Drupal cache.
-   *
-   * @inheritdoc
+   * {@inheritdoc}
    */
-  public function save($id, $data, $lifeTime = NULL) {
+  protected function doSave($id, $data, $lifeTime = 0) {
     // Doctrine defines NULL to mean permanent cache.
-    if ($lifeTime == NULL) {
+    if ($lifeTime === NULL) {
```

Comment:
I originally did that, but then it turns out that if [you explicitly pass NULL in as $lifetime](https://php.net/manual/en/functions.arguments.php#70511) that the default argument doesn't get set. That's what [this test covers](https://github.com/Capgemini/drupal-doctrine-cache/pull/2/files#diff-a38057b53968471fe9b75cb89e824562R31).